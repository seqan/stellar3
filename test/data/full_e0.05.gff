1	Stellar	eps-matches	1242	1293	96.1538	+	.	1;seq2Range=719,770;eValue=2.81906e-18;cigar=52M;mutations=40C,44T
1	Stellar	eps-matches	127	177	96.0784	+	.	1;seq2Range=880,930;eValue=9.26648e-18;cigar=51M;mutations=22A,38A
1	Stellar	eps-matches	184	234	96.0784	+	.	1;seq2Range=1090,1140;eValue=9.26648e-18;cigar=51M;mutations=7G,27T
1	Stellar	eps-matches	729	779	96.0784	+	.	1;seq2Range=662,711;eValue=9.26648e-18;cigar=49M1D1M;mutations=13C
1	Stellar	eps-matches	1247	1296	96	+	.	1;seq2Range=595,644;eValue=3.04597e-17;cigar=50M;mutations=38A,46A
1	Stellar	eps-matches	1371	1422	96.1538	-	.	1;seq2Range=1600,1651;eValue=2.81906e-18;cigar=52M;mutations=3G,25G
1	Stellar	eps-matches	1538	1587	96	-	.	1;seq2Range=1723,1772;eValue=3.04597e-17;cigar=50M;mutations=20G,31C
1	Stellar	eps-matches	1104	1153	96	-	.	1;seq2Range=1859,1908;eValue=3.04597e-17;cigar=50M;mutations=5A,42T
1	Stellar	eps-matches	744	793	96	-	.	1;seq2Range=1978,2027;eValue=3.04597e-17;cigar=50M;mutations=29C,42C
1	Stellar	eps-matches	486	535	96	-	.	1;seq2Range=1296,1345;eValue=3.04597e-17;cigar=50M;mutations=41A,47G
