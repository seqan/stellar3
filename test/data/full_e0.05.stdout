I/O options:
  database file   : ref.fasta
  query file      : query_e0.05.fasta
  alphabet        : dna5
  output file     : full_e0.05.gff
  output format   : gff

User specified parameters:
  minimal match length             : 50
  maximal error rate (epsilon)     : 0.05 = (1/20)
  maximal x-drop                   : 5
  search forward strand            : yes
  search reverse complement        : yes

  verification strategy            : exact
  maximal number of matches        : 50
  duplicate removal every          : 500

Calculated parameters:
  k-mer length : 15
  s^min        : 15
  threshold    : 1
  distance cut : 45
  delta        : 16
  overlap      : 2

Loaded 1 query sequence.
Loaded 1 database sequence.

All matches resulting from your search have an E-value of: 
        3.04597e-17 or smaller  (match score = 1, error penalty = -2)

Constructing index...

Aligning all query sequences to database sequence...
  1
    # SWIFT hits      : 5
    Longest hit       : 44
    Avg hit length    : 38
  1, complement
    # SWIFT hits      : 5
    Longest hit       : 50
    Avg hit length    : 40

# Eps-matches     : 10
Longest eps-match : 52
Avg match length  : 50
